traffic_ops/client/topology.go

```diff
@@ -30,94 +30,118 @@ const (
 )
 
 // CreateTopology creates a topology and returns the response.
-func (to *Session) CreateTopology(top tc.Topology) (*tc.TopologyResponse, ReqInf, error) {
-	var remoteAddr net.Addr
+func (to *Session) CreateTopology(top tc.Topology) (*tc.TopologyResponse, ReqInf, error, int) {
+	var (
+		statusCode = http.StatusNotAcceptable
+		remoteAddr net.Addr
+	)
 	reqBody, err := json.Marshal(top)
 	reqInf := ReqInf{CacheHitStatus: CacheHitStatusMiss, RemoteAddr: remoteAddr}
 	if err != nil {
-		return nil, reqInf, err
+		return nil, reqInf, err, statusCode
```

Comment:
Based on the mailing list discussion it sounds like consensus has been built around adding the status code to the `ReqInf` struct and returning it that way, so we should start following that pattern for these methods. Then in a separate PR someone could retroactively add the status code to `ReqInf` for the rest of the TO Go client methods.

Reply:
Following that pattern in c1140e42f4

Reply:
Did you not want to start setting the status code in `ReqInf` in these Topologies methods? Or that likely means having to update all of its usage throughout the entire TO client?

Reply:
Let's do it. Setting the status code in 481038d58a.